packages/e2e-test-utils/src/index.js

```diff
@@ -15,6 +15,7 @@ export { enablePrePublishChecks } from './enable-pre-publish-checks';
 export { ensureSidebarOpened } from './ensure-sidebar-opened';
 export { findSidebarPanelToggleButtonWithTitle } from './find-sidebar-panel-toggle-button-with-title';
 export { findSidebarPanelWithTitle } from './find-sidebar-panel-with-title';
+export { getAllBlockInserterItemTitles } from './get-all-block-inserter-item-titles';
```

Comment:
Can we also update CHANGELOG file? This package is public and is already being explored by external projects. This is going to be minor version bump as we add new features 🎉